spec/mailers/provider_mailer_spec.rb

```diff
@@ -429,6 +429,11 @@
   end
 
   describe 'apply_service_is_now_open' do
+    before do
+      allow(CycleTimetable).to receive(:apply_opens).and_return(Date.new(2021, 10, 12))
+      allow(CycleTimetable).to receive(:cycle_year_range).and_return('2022 to 2023')
```

Comment:
Would `Timecop.travel` provide a more realistic setup here?

Reply:
agree – basically was a C&P of the candidate tests

Reply:
It's probably better to be code-consistent throughout the test than to amend one piece of setup. It's really not a blocker.

Reply:
timecopping